pkg/generator/golang_generator_test.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2023 API Testing Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package generator
+
+import (
+	"net/http"
+	"strings"
+	"testing"
+
+	atest "github.com/linuxsuren/api-testing/pkg/testing"
+)
+
+func TestGolangGenerator_Generate(t *testing.T) {
+	generator := NewGolangGenerator()
+
+	testSuite := &atest.TestSuite{
+		Name: "Test Suite Example",
+	}
+
+	testcase := &atest.TestCase{
+		Name: "Test Case Example",
+		Request: atest.Request{
+			Method: "POST",
+			API:    "http://127.0.0.1:8082/metrics",
```

Comment:
you can use https://httpbin.org

for example: `https://httpbin.org/get`

Reply:
Actually, all unit tests should not rely on any other services. Because it might run in an environment without internet.